source/webhook-trigger.py

```diff
@@ -116,10 +116,9 @@ def verify_environment(variables):
 
 
 if sys.version_info < MIN_PYTHON:
-    error_message = "Found Python " + str(sys.version_info) + " but Python %s.%s or later is required. " \
-                                                              "Unable to find towel. Panicking.\n" % MIN_PYTHON
-    send_slack_error_message(error_message, None)
-    sys.exit(error_message)
+    build_exception_message = "Found Python " + str(sys.version_info) + " but need Python %s.%s or later." % MIN_PYTHON
```

Comment:
I'd keep the name of this `exception_message` as all you're doing is building a string. My comment about renaming in the other PR was about the function name not the variable name

Reply:
`exception_message` would conflict with the function of the same name. Would you be happy with just `message`?

Reply:
`send_slack_error_message` does send a message to slack

Reply:
Sorry, I meant `error_message`. I.e. leave the name as it was

Reply:
i have moved this into a function and renamed the var to `message` to align with the other functions. Is that OK?